build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -217,12 +217,19 @@ tlsMinVersion: {{ .Values.tlsMinVersion | quote }}
 # 3. The Node IP
 transportInterface: {{ .Values.transportInterface | quote }}
 
+multicast:
+{{- with .Values.multicast }}
 # The names of the interfaces on Nodes that are used to forward multicast traffic.
 # Defaults to transport interface if not set.
-multicastInterfaces:
-{{- with .Values.multicastInterfaces }}
-{{- toYaml . | nindent 2 }}
-{{- end }}
+  multicastInterfaces:
+  {{- with .multicastInterfaces }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+
+# The interval at which the antrea-agent to send IGMP queries to Pods.
```

Comment:
s/to send/sends

in all the other places as well

Reply:
done